ccv/models.py

```diff
@@ -877,27 +888,29 @@ class CommitteeMembership(Base):
                                    help_text="Description of services contributed by the person as part of a committee")
     end_date = models.DateField(null=True, blank=True, help_text="The date on which membership ended, if applicable")
 
-    organization = models.OneToOneField(Organization, null=True, blank=True, on_delete=models.DO_NOTHING,
+    organization = models.OneToOneField(Organization, null=True, blank=True, on_delete=models.CASCADE,
                                         help_text="The name of the organisation of which the person is a member")
-    other_organization = models.OneToOneField(OtherOrganization, null=True, blank=True, on_delete=models.DO_NOTHING)
-    membership = models.ForeignKey(Membership, on_delete=models.DO_NOTHING)
+    other_organization = models.OneToOneField(OtherOrganization, null=True, blank=True, on_delete=models.CASCADE)
+    membership = models.ForeignKey(Membership, on_delete=models.CASCADE)
 
 
 class OtherMembership(Base):
     """Services contributed as part of a scholarly society or other organization to perform services not directly
     related to the person's research activities """
 
-    role = models.CharField(max_length=20, null=True, blank=True, help_text="The person's role in this activity")
+    role = models.CharField(max_length=DEFAULT_COLUMN_LENGTH, null=True, blank=True, help_text="The person's role in this activity")
     name = models.CharField(max_length=250, null=True, blank=True, help_text="The name of the committee")
     start_date = models.DateField(null=True, blank=True, help_text="The date on which membership began")
     description = models.CharField(max_length=1000, null=True, blank=True,
                                    help_text="Description of services contributed by the person as part of a committee")
     end_date = models.DateField(null=True, blank=True, help_text="The date on which membership ended, if applicable")
 
-    organization = models.OneToOneField(Organization, null=True, blank=True, on_delete=models.DO_NOTHING,
+    organization = models.OneToOneField(Organization, null=True, blank=True, on_delete=models.CASCADE,
                                         help_text="The name of the organisation of which the person is a member")
-    other_organization = models.OneToOneField(OtherOrganization, null=True, blank=True, on_delete=models.DO_NOTHING)
-    membership = models.ForeignKey(Membership, on_delete=models.DO_NOTHING)
+    other_organization = models.OneToOneField(OtherOrganization, null=True, blank=True, on_delete=models.CASCADE)
+    membership = models.ForeignKey(Membership, on_delete=models.CASCADE)
+
+########################################################################################################################
```

Comment:
With the file at 2400 lines I think it's too much, like we've discuss some time ago, I believe it'd be better to divide it in 2-3 files or so. Like models/user_profile.py etc..